pages/index.tsx

```diff
@@ -46,26 +47,18 @@ export const HomePage: NextPage = () => {
   const [selectedNeighborhoods, setSelectedNeighborhoods] = useState<string[]>(
     []
   )
-  const [resourceSortLabel, setResourceSortLabel] =
-    useState<string>('Sort Ascending')
-
-  const [resourceSortMethod, setResourceSortMethod] =
-    useState<ResourceSortMethod>('Last Modified')
-  const [resourceSortAscending, setResourceSortAscending] =
-    useState<boolean>(false)
-  const changeResourceSortOrder = () => {
-    setResourceSortLabel(
-      resourceSortAscending ? 'Sort Ascending' : 'Sort Descending'
-    )
-    setResourceSortAscending(!resourceSortAscending)
-  }
 
   const fuse = useMemo(() => {
     return new Fuse(allResources ?? [], { keys: RESOURCE_SEARCH_FIELDS })
   }, [allResources])
   // For keyword searches
   const [keyword, setKeyword] = useState('')
 
+  const [resourceSortMethod, setResourceSortMethod] = useState<
+    ResourceSortMethod | undefined
```

Comment:
Making the sort method `undefined` by default because with it set, when you search, the most relevant results don't show up first which I think would be confusing for users unless the sorted after searching